uwblueprint/paramedics-react
# Patient Cleanup

[Notion ticket](https://www.notion.so/uwblueprintexecs/396370c5e09f4c28bc7f455a7e84c908?v=3751aa62a2594c74b7e0ecb363304c24&p=7108d3eee3cf4c96893e3de358ae5a55)

## Changes
- Fixed 2/3 `useEffect` errors, but this will be addressed in a later ticket
- Checked that routing after cancel or submitting is working properly 
- Made patient status un-toggable (a user cannot have a empty status unless previously deleted)
- Added mandatory validation for age

## Screenshots

## Testing
- Check that routing is working correctly: any submission (deletion/completion) or cancelling should take you to the CCP dashboard
- Make sure that status buttons are not toggalable (as in, if you were to select the status for one patient, you cannot click again and assign no status)
- Try to submit a patient with no age. The submission should not work and an appropriate message should show.

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
